Ensure that correct config filepath is printed when initializing character (#2617)
nucypher/nucypher: docs/source/staking/running_a_worker.rst

    $(nucypher) pip install -U nucypher

> 2. Configure the worker using the nucypher CLI, and store the configuration to disk:

Author: ✔️

   * ``<GWEI>`` - The maximum price of gas to spend on commitment transactions

.. important::

    The default configuration file is ``ursula.json``. If there is an existing default configuration file, a new

>     Worker configuration file with a unique suffix is created e.g. ``ursula-0216ad10.json``. Since this file would not

Author: ✔️

   * ``<GWEI>`` - The maximum price of gas to spend on commitment transactions

The configuration settings will be stored in an ursula configuration file.

.. important::

    The default configuration file is ``ursula.json``. If there is an existing default configuration file, a new

>     Worker configuration file suffixed by the first 8 characters of the node's signing public key

Reviewer: For simplification perhaps just use the wording "the node's public key" - We can start to suggest a convention, afterall this is the core element and invariant of the node's identity.

Author: ✔️

    e.g. ``ursula-0216ad10.json``. Since this file is not the default, subsequent ``ursula`` CLI commands
    can use the ``--config-file <FILEPATH>`` option to specify the non-default filepath of the Worker

>     configuration file.

Reviewer: Mention iterative selection as well?

Author: ✔️

   * ``<GWEI>`` - The maximum price of gas to spend on commitment transactions

The configuration settings will be stored in an ursula configuration file.

.. important::

    The default configuration file is ``ursula.json``. If there is an existing default configuration file, a new
    Worker configuration file suffixed by the first 8 characters of the node's signing public key

>     e.g. ``ursula-0216ad10.json``. Since this file would not be the default, subsequent ``ursula`` CLI commands
>     should use the ``--config-file <FILEPATH>`` option to specify the non-default filepath of the Worker

Reviewer: ```suggestion
  e.g. ``ursula-0216ad10.json``. Since this file is not the default, subsequent ``ursula`` CLI commands
    can use the ``--config-file <FILEPATH>`` option to specify the non-default filepath of the Worker
```

Author: ✔️

    configuration file.

.. note::

    All worker configuration values can be changed using the `config` command

>     (Note that the worker must be restarted for new changes to take effect):

Reviewer: I think this note deserves it's own note section.

Author: ✔️

    .. code::

        nucypher ursula config --<OPTION> <NEW VALUE>

        # Update the max gas price setting
        nucypher ursula config --max-gas-price <GWEI>

        # Update the max gas price setting
        nucypher ursula config --provider <PROVIDER URI>

        # View the current configuration
        nucypher ursula config

> 

Reviewer: ```suggestion 
        # View the current configuration of a non-default ursula
        nucypher ursula config --config-file <CONFIG PATH>

   * ``<SIGNER URI>`` - The URI to an ethereum keystore or signer: `keystore:///root/.ethereum/keystore`
   * ``<GWEI>`` - The maximum price of gas to spend on commitment transactions

> The configuration settings will be stored in an ursula configuration file.

Reviewer 2: Minor detail - capitalize Ursula here

Author: ✔️

   * ``<SIGNER URI>`` - The URI to an ethereum keystore or signer: `keystore:///root/.ethereum/keystore`
   * ``<GWEI>`` - The maximum price of gas to spend on commitment transactions

The configuration settings will be stored in an ursula configuration file.

.. important::

    The default configuration file is ``ursula.json``. If there is an existing default configuration file, a new
    Worker configuration file suffixed by the first 8 characters of the node's signing public key

>     e.g. ``ursula-0216ad10.json``. Since this file would not be the default, subsequent ``ursula`` CLI commands
>     should use the ``--config-file <FILEPATH>`` option to specify the non-default filepath of the Worker

Reviewer: ```suggestion
  e.g. ``ursula-0216ad10.json``. Since this file is not the default, subsequent ``ursula`` CLI commands
    can use the ``--config-file <FILEPATH>`` option to specify the non-default filepath of the Worker
```

Author: ✔️

   * ``<SIGNER URI>`` - The URI to an ethereum keystore or signer: `keystore:///root/.ethereum/keystore`

Update Worker Configuration
^^^^^^^^^^^^^^^^^^^^^^^^^^^

All worker configuration values can be modified using the `config` command. For non-default worker configuration file
paths, use the ``--config-file <CONFIG PATH>`` parameter.

.. code::

    #
    # Default configuration file path
    #

    nucypher ursula config --<OPTION> <NEW VALUE>

    # Update the max gas price setting
    nucypher ursula config --max-gas-price <GWEI>

    # Change the Ethereum provider to use
    nucypher ursula config --provider <PROVIDER URI>

    # View the current configuration
    nucypher ursula config

    #
    # Non-default configuration file path
    #

    # View the current configuration of a non-default configuration file path
    nucypher ursula config --config-file <CONFIG PATH>

    # Update the max gas price setting of a non-default configuration file path
    nucypher ursula config --config-file <CONFIG PATH> --provider <PROVIDER URI>

.. important::

>     The worker must be restarted for new changes to take effect

Reviewer: ```suggestion
    The worker must be restarted for new changes to take effect.
```

Author: ✔️
